domain/keyupdater/state/types.go

```diff
@@ -3,8 +3,8 @@
 
 package state
 
-// authorisedKey represents a single authorised key for a machine.
-type authorisedKey struct {
+// authorizedKey represents a single authorised key for a machine.
+type authorizedKey struct {
```

Comment:
Can we use British spelling - not sure why the change here as British is our standard.

Reply:
because SSH uses `authorize` and it relates to that. We kept this convention in the original code so I have kept it going.